### PR TITLE
doc: replace LDADD -> LDFLAGS in OSX instructions

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

**Comment by gperciva** [comment]
Sorry, I forgot to change one instance of `LDADD` when I copied these from the spiped `BUILDING` file.

File: BUILDING [hunk 1]
```diff
@@ -18,11 +18,14 @@ Platform-specific notes
   10.11 "El Capitan" and higher, OpenSSL was removed entirely.  After
   installing a newer version of OpenSSL, use
       CPPFLAGS="-I /path/to/openssl/headers"
-      LDADD="-L /path/to/openssl/lib"
+      LDFLAGS="-L /path/to/openssl/lib"
   to build spiped.
```
Comment (cperciva) [inline BUILDING]: It occurs to me that we should probably have s/spiped/scrypt/ here too...

Comment (gperciva) [inline BUILDING]: whoops.  I also discovered another "spiped" earlier in that paragraph.

**Comment by gperciva** [comment]
PR updated.  I think it's decent now, but I wrecked myself at the gym over lunch so I wouldn't trust my ability to proof-read anything over a grade 2 reading level at the moment.  :)

File: BUILDING [hunk 2]
```diff
@@ -14,15 +14,18 @@ Platform-specific notes
...
-  to build spiped.
+      LDFLAGS="-L /path/to/openssl/lib"
+  to build scrypt.
 
   In particular, if you installed OpenSSL using homebrew, you may
-  add these lines to your $HOME/.profile file:
+  pass the relevant directories directly to ./configure:
+      ./configure CPPFLAGS="-I/usr/local/opt/openssl/include" LDFLAGS="-L/usr/local/opt/openssl/lib"
+
+  Alternately, you may with to add these lines to your $HOME/.profile file:
```
Comment (cperciva) [inline BUILDING]: Alternatively.  The use of "alternately" to mean "alternatively" is an American perversion.

Comment (gperciva) [inline BUILDING]: well, I don't think either of those is on the grade 2 reading list.  :)

**Comment by gperciva** [comment]
updated PR

**Pull request merged**